src/en/community/events/2023/ceph-days-korea/index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Ceph Days Korea
+date: 2023-06-16
+end: 2023-06-16
+location: Seoul, South Korea Room Sae-gae-ro, AT Center, 27, Gangnam-daero, Seocho-gu
+venue: AT 센터
+image: "/assets/bitmaps/ceph-days.png"
+tags:
+  - ceph days
+---
+
+### Save the Date - Ceph is coming to Korea!
+
+A full-day event dedicated to sharing Ceph’s transformative power and fostering
+the vibrant Ceph community in South Korea
+
+The expert Ceph team, Ceph’s customers and partners, and the Ceph community
+join forces to discuss things like the status of the Ceph project, recent Ceph
+project improvements and roadmap, and Ceph community news. The day ends with
+a networking reception, to foster more Ceph learning.
+
+<a class="button" href="#">CFP Coming Soon</a>
+<a class="button" href="#">Registeration Coming Soon</a>
+
+## Important Dates
+
+- CFP Opens: Coming Soon!
+- CFP Closes: 2023-04-28
+- Speakers receive confirmation of acceptance: 2023-05-12
+- Schedule Announcement: 2023-05-16
+- Sponsorship Deadline: 2023-04-28
+- Event Date: 2023-06-16
```

Comment:
Sorry, Event Data is 2023-06-14 (KST)

Reply:
@myoungwon Updated the date. Please review again. @zdover23, please merge after.

Reply:
LGTM